### PR TITLE
Add avoidManeuverSeconds to RouteAlternativesOptions

### LINKED ISSUE #5393: RouteAlternativesController options
RouteAlternativesController should not use hardcoded
`MIN_TIME_BEFORE_MANEUVER_SECONDS`, it should use `RerouteOptions::avoidManeuverSeconds`
or we should add the same property to `RouteAlternativesOptions` and use it

### PR DESCRIPTION
**Pull request by korshaknn**
### Description

closes #5393

### CODE REVIEW

File: libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteAlternativesOptions.kt [hunk 1]
```diff
@@ -62,30 +70,43 @@ class RouteAlternativesOptions private constructor(
      */
     class Builder {
         private var intervalMillis: Long = TimeUnit.MINUTES.toMillis(5)
+        private var avoidManeuverSeconds = 1
```
Comment (RingerJK) [inline libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteAlternativesOptions.kt:73]: let's set to `8`, it's recomended value 
```suggestion
        private var avoidManeuverSeconds = 8
```

Comment (korshaknn) [reply libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteAlternativesOptions.kt:73]: where is it recommended? hardcoded value was `1`, NN has `0` as  default

Comment (RingerJK) [reply libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteAlternativesOptions.kt:73]: it's the recommendation of automotive [doc](https://docs.google.com/document/d/1y3Aqj3I5rfLBHyyfusO9RwV8lH0G4ckMI_xJKBLfFts/edit#)
```
How should it be implemented?
We will use the speed of the edge to determine the vehicle speed. The next maneuver must not be earlier than 8s ahead (best case: configurable) based on the current speed.
```

**Comment by korshaknn** [comment]
set `MINIMUM_REQUEST_ALTERNATIVES_INTERVAL` to 30sec as required by NavNative

**Review by RingerJK — APPROVED** [review]

**Review by VysotskiVadim — APPROVED** [review]

**Pull request merged**